source/common/http/http1/codec_impl.cc

```diff
@@ -435,16 +423,16 @@ void ResponseEncoderImpl::encodeHeaders(const ResponseHeaderMap& headers, bool e
   encodeHeadersBase(headers, absl::make_optional<uint64_t>(numeric_status), end_stream, false);
 }
 
-static const char REQUEST_POSTFIX[] = " HTTP/1.1\r\n";
+static constexpr absl::string_view REQUEST_POSTFIX = " HTTP/1.1\r\n";
 
 Status RequestEncoderImpl::encodeHeaders(const RequestHeaderMap& headers, bool end_stream) {
   // Required headers must be present. This can only happen by some erroneous processing after the
   // downstream codecs decode.
   RETURN_IF_ERROR(HeaderUtility::checkRequiredRequestHeaders(headers));
 
   const HeaderEntry* method = headers.Method();
-  const HeaderEntry* path = headers.Path();
-  const HeaderEntry* host = headers.Host();
+  const HeaderEntry* path = headers.Path(); // NOLINT(clang-analyzer-core.CallAndMessage)
```

Comment:
why is this NOLINT pragma needed? What does clang-tidy say if we don't have this?

Reply:
I forgot it, and I will check it afterwards.